src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -2393,7 +2393,6 @@ internal bool ContainsNoPiaLocalTypes()
             return _lazyContainsNoPiaLocalTypes == ThreeState.True;
         }
 
-        // PROTOTYPE(NullableReferenceTypes): Remove if not needed.
         internal bool UtilizesNullableReferenceTypes()
```

Comment:
Not related to this PR: I found the name of this method (and properties with same name) confusing. This doesn't mean that there are any nullable reference types, but rather that this is a "modern" assembly, compiled with "nullable" feature turned on.

Reply:
This assembly could have been created in some other tool than the C# compiler though, so it's not strictly related to the (C#) feature.

---
In reply to: [171072311](https://github.com/dotnet/roslyn/pull/25087#discussion_r171072311) [](ancestors = 171072311)